install/helm/agones/templates/controller.yaml

```diff
@@ -120,6 +120,8 @@ spec:
           valueFrom:
             fieldRef:
               fieldPath: metadata.name
+        - name: DISABLE_MTLS
```

Comment:
We don't need this here.

Reply:
Removed mTLS disabling from the gameserverallocator since we left that as a TODO for multi-cluster.